pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -159,9 +159,11 @@ private void preGenesisSubscription(final BigInteger lastProcessedBlockNumber) {
 
   private SafeFuture<EthBlock.Block> sendDepositsUpToMinGenesis(
       final EthBlock.Block minGenesisTimeBlock, final ReplayDepositsResult replayDepositsResult) {
+    final BigInteger startBlock =
+        getFirstUnprocessedBlockNumber(replayDepositsResult)
+            .max(minimumGenesisTimeBlockFinder.getMinFirstDepositBlock().bigIntegerValue());
```

Comment:
Could have sworn I posted this a moment ago so it may turn up twice but, `getFirstUnprocessedBlockNumber` already takes into account the `minFirstDepositBlock` so I don't think we need this change. Or am I missing something?

Reply:
Oh, you're right!  I read past that somehow.